### DISCUSSION TITLE
Issue when registering a new account

### DISCUSSION THREAD

**Discussion by makver9** [OktopUSP/oktopus#322] [Q&A]
I have cloned the repo and used compose command to run the containers. 

The UI opens fine and I want to register a new user because I just can't login through, 

On registering I see an error as highlighted in the below screenshot. Please help

![image](https://github.com/user-attachments/assets/97f18cfa-dfc2-43b8-a20c-e560617ef33d)

**Comment by younes-medhioub** [comment]
Hello makver9,
I just wanted some more details. is mongo container up ?
if No , try edit docker-compose.yaml:
image: mongo  -> image: mongo:4.4.6

**Comment by makver9** [comment]
Hey @younes-medhioub , 

Thank you for your reply, 

Dumping the output of docker ps  here


CONTAINER ID   IMAGE                           COMMAND                  CREATED         STATUS         PORTS                                                                                            NAMES
fa279b5c474a   oktopusp/stomp-adapter          "/stomp-adapter"         2 minutes ago   Up 2 minutes                                                                                                    stomp-adapter
ab4dd616a8f7   oktopusp/ws-adapter             "/ws-adapter"            2 minutes ago   Up 2 minutes                                                                                                    ws-adapter
49306aaeb8e5   oktopusp/controller             "/controller"            2 minutes ago   Up 2 minutes   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp                                                        controller
18da6edc09ce   oktopusp/adapter                "/adapter"               2 minutes ago   Up 2 minutes                                                                                                    adapter
00cb270a5115   oktopusp/mqtt-adapter           "/mqtt-adapter"          2 minutes ago   Up 2 minutes                                                                                                    mqtt-adapter
8eb3e2cf664b   portainer/portainer-ce:latest   "/portainer"             2 minutes ago   Up 2 minutes   8000/tcp, 9000/tcp, 0.0.0.0:9443->9443/tcp, :::9443->9443/tcp                                    portainer
b35368007591   oktopusp/ws                     "/ws"                    2 minutes ago   Up 2 minutes   0.0.0.0:8080->8080/tcp, :::8080->8080/tcp                                                        websockets
2987f55e21dd   nats:latest                     "/nats-server -c /tm…"   2 minutes ago   Up 2 minutes   0.0.0.0:4222->4222/tcp, :::4222->4222/tcp, 0.0.0.0:8222->8222/tcp, :::8222->8222/tcp, 6222/tcp   nats
f33564968a32   oktopusp/acs                    "/acs"                   2 minutes ago   Up 2 minutes   0.0.0.0:9292->9292/tcp, :::9292->9292/tcp                                                        acs
02714b4b3afc   oktopusp/file-server            "/file-server"           2 minutes ago   Up 2 minutes   0.0.0.0:8004->8004/tcp, :::8004->8004/tcp                                                        file-server
6642744c9dc9   oktopusp/socketio               "docker-entrypoint.s…"   2 minutes ago   Up 2 minutes   0.0.0.0:5000->5000/tcp, :::5000->5000/tcp                                                        socketio
2e47c9452104   oktopusp/mqtt                   "/mqtt"                  2 minutes ago   Up 2 minutes   0.0.0.0:1883->1883/tcp, :::1883->1883/tcp, 0.0.0.0:8883->8883/tcp, :::8883->8883/tcp             mqtt
7a4b43dad549   mongo:4.4.6                     "docker-entrypoint.s…"   2 minutes ago   Up 2 minutes   0.0.0.0:27017->27017/tcp, :::27017->27017/tcp                                                    mongo_usp
55ee21a310a4   oktopusp/stomp                  "/stomp"                 2 minutes ago   Up 2 minutes   0.0.0.0:61613->61613/tcp, :::61613->61613/tcp                                                    stomp
51d71fbae990   nginx:latest                    "/docker-entrypoint.…"   2 minutes ago   Up 2 minutes   0.0.0.0:80->80/tcp, :::80->80/tcp                                                                nginx
312b2fc83760   oktopusp/frontend-ce            "docker-entrypoint.s…"   2 minutes ago   Up 2 minutes   0.0.0.0:3000->3000/tcp, :::3000->3000/tcp                                                        frontend
 
 
 same behaviour with mongo 4.4.6 as well
